src/overlays/actors/ovl_Obj_Hgdoor/z_obj_hgdoor.c

```diff
@@ -18,13 +18,13 @@ void ObjHgdoor_Draw(Actor* thisx, PlayState* play);
 
 void ObjHgdoor_SetupCheckShouldOpen(ObjHgdoor* this);
 void ObjHgdoor_CheckShouldOpen(ObjHgdoor* this, PlayState* play);
-void func_80BD42AC(ObjHgdoor* this);
-void func_80BD42C0(ObjHgdoor* this, PlayState* play);
-void func_80BD433C(ObjHgdoor* this);
-void func_80BD4358(ObjHgdoor* this, PlayState* play);
-void func_80BD4460(ObjHgdoor* this);
-void func_80BD4478(ObjHgdoor* this, PlayState* play);
-s32 func_80BD44D0(ObjHgdoor* this, PlayState* play);
+void ObjHgdoor_SetupCutscene(ObjHgdoor* this);
+void ObjHgdoor_PlayCutscene(ObjHgdoor* this, PlayState* play);
+void ObjHgdoor_SetupCsAction(ObjHgdoor* this);
+void ObjHgdoor_HandleCsAction(ObjHgdoor* this, PlayState* play);
+void ObjHgdoor_SetupStopCs(ObjHgdoor* this);
+void ObjHgdoor_StopCs(ObjHgdoor* this, PlayState* play);
+s32 ObjHgdoor_RotateDoor(ObjHgdoor* this, PlayState* play);
```

Comment:
This might just be me (i.e. feel free to leave it as it is if you want), but personally the `Door`  in `ObjHgdoor_RotateDoor`, `sDoorOpenFlag`, and `ObjHgdoor_OepnDoors` seem redundant and not needed, since the actor themselves are the doors.

Reply:
That's fair, when I was naming these it was when I was looking at it in reference to pamela, pamela's dad and other things in the scene. I agree on the redundancy